### PR TITLE
[Rule Tuning] Windows DR Tuning - 12

### PR DESCRIPTION
**Pull request by w0rk3r**
## Issues

Part of #3186 
Resolves https://github.com/elastic/detection-rules/issues/3103

## Summary

Tunes the following rules:

* Suspicious MS Office Child Process
* Direct Outbound SMB Connection
* Remote Execution via File Shares
* Remote File Copy to a Hidden Share
* Remotely Started Services via RPC

### CODE REVIEW

File: rules/windows/lateral_movement_remote_file_copy_hidden_share.toml [hunk 1]
```diff
@@ -34,8 +34,11 @@ type = "eql"
 
 query = '''
 process where host.os.type == "windows" and event.type == "start" and
-  process.name : ("cmd.exe", "powershell.exe", "robocopy.exe", "xcopy.exe") and
-  process.args : ("copy*", "move*", "cp", "mv") and process.args : "*$*"
+  (
+    process.name : ("cmd.exe", "powershell.exe", "xcopy.exe") and
+    process.args : ("copy*", "move*", "cp", "mv") or
+    process.name : "robocopy.exe"
+  ) and process.args : "*\\\\*\\*$*"
```
Comment (w0rk3r) [inline rules/windows/lateral_movement_remote_file_copy_hidden_share.toml:41]: The `process.args : "*$*"` condition is causing a lot of unintended matches, as the `$` chars are heavily used for other purposes like variables in PowerShell and file names. I am replacing it with a pattern that should match only hidden shares.

File: rules/windows/lateral_movement_remote_services.toml [hunk 2]
```diff
@@ -114,35 +114,36 @@ sequence with maxspan=1s
    ] by host.id, process.entity_id
    [process where host.os.type == "windows" and 
        event.type == "start" and process.parent.name : "services.exe" and
-       not (process.executable : "?:\\Windows\\System32\\svchost.exe" and process.args : "tiledatamodelsvc") and
```
Comment (w0rk3r) [inline rules/windows/lateral_movement_remote_services.toml:117]: Removing this as svchost is entirely excluded in the process.executable below

File: rules/windows/lateral_movement_remote_services.toml [hunk 3]
```diff
@@ -114,35 +114,36 @@ sequence with maxspan=1s
...
+       not process.executable : (
+                "?:\\Pella Corporation\\OSCToGPAutoService\\OSCToGPAutoSvc.exe",
+                "?:\\Pella Corporation\\Pella Order Management\\GPAutoSvc.exe",
+                "?:\\Pella Corporation\\Pella Order Management\\GPAutoSvc.exe",
                 "?:\\Program Files (x86)\\*.exe",
                 "?:\\Program Files\\*.exe",
+                "?:\\Windows\\ADCR_Agent\\adcrsvc.exe",
+                "?:\\Windows\\AdminArsenal\\PDQ*.exe",
+                "?:\\Windows\\CAInvokerService.exe",
+                "?:\\Windows\\ccmsetup\\ccmsetup.exe",
```
Comment (w0rk3r) [inline rules/windows/lateral_movement_remote_services.toml:127]: Actual change, rest is sort.

File: rules/windows/lateral_movement_remote_services.toml [hunk 4]
```diff
@@ -114,35 +114,36 @@ sequence with maxspan=1s
...
+                "?:\\Windows\\RemoteAuditService.exe",
+                "?:\\Windows\\servicing\\TrustedInstaller.exe",
+                "?:\\Windows\\System32\\certsrv.exe",
                 "?:\\Windows\\System32\\sppsvc.exe",
-                "?:\\Windows\\System32\\wbem\\WmiApSrv.exe",
-                "?:\\WINDOWS\\RemoteAuditService.exe",
-                "?:\\Windows\\VeeamVssSupport\\VeeamGuestHelper.exe",
-                "?:\\Windows\\VeeamLogShipper\\VeeamLogShipper.exe",
-                "?:\\Windows\\CAInvokerService.exe",
+                "?:\\Windows\\System32\\srmhost.exe",
```
Comment (w0rk3r) [inline rules/windows/lateral_movement_remote_services.toml:135]: Actual change, rest is sort.

File: rules/windows/initial_access_suspicious_ms_office_child_process.toml [hunk 5]
```diff
@@ -84,14 +84,26 @@ type = "eql"
...
+      "installutil.exe", "ipconfig.exe", "mshta.exe", "msxsl.exe", "nbtstat.exe", "net.exe", "net1.exe", "netsh.exe",
+      "netstat.exe", "nltest.exe", "odbcconf.exe", "ping.exe", "powershell.exe", "pwsh.exe", "qprocess.exe",
+      "quser.exe", "qwinsta.exe", "rcsi.exe", "reg.exe", "regasm.exe", "regsvcs.exe", "regsvr32.exe", "sc.exe",
+      "schtasks.exe", "systeminfo.exe", "tasklist.exe", "tracert.exe", "whoami.exe", "wmic.exe", "wscript.exe",
+      "xwizard.exe", "explorer.exe", "rundll32.exe", "hh.exe", "msdt.exe"
+  ) and
+  not (
+    process.parent.name : "outlook.exe" and
+    process.name : "rundll32.exe" and
+    process.args : "shell32.dll,Control_RunDLL" and
```
Comment (terrancedejesus) [inline rules/windows/initial_access_suspicious_ms_office_child_process.toml:104]: Q: Is this how the side DLL and entry method are parsed specifically or do we need wildcards?

File: rules/windows/lateral_movement_direct_outbound_smb_connection.toml [hunk 6]
```diff
@@ -104,21 +104,14 @@ query = '''
 sequence by process.entity_id with maxspan=2m
   [process where host.os.type == "windows" and event.type == "start" and process.pid != 4 and 
    not user.id : ("S-1-5-19", "S-1-5-20") and 
-   not (process.code_signature.trusted == true and not process.code_signature.subject_name : ("Microsoft*", "Famatech Corp.", "Insecure.Com LLC")) and
```
[the remark this answers is not in the document]
Comment (terrancedejesus) [reply rules/windows/lateral_movement_direct_outbound_smb_connection.toml:107]: This rule has been the subject of recent questions based on performance. If we remove this from the first sequence for legitimate binaries, the events captured could be exponential. Just something to consider.

File: rules/windows/initial_access_suspicious_ms_office_child_process.toml (hunk 5 above)
Comment (w0rk3r) [reply rules/windows/initial_access_suspicious_ms_office_child_process.toml:104]: image

File: rules/windows/lateral_movement_remote_file_copy_hidden_share.toml (hunk 1 above)
Comment (terrancedejesus) [reply rules/windows/lateral_movement_remote_file_copy_hidden_share.toml:41]: Performance wise, does it make sense to rely on EQL's built in regex?

`process.args regex~ "\\\\[^\s\\]+\\[^\s\\]+\$.*"

**Review by terrancedejesus — APPROVED** [review]
Some questions added. Looks good though!

File: rules/windows/lateral_movement_remote_file_copy_hidden_share.toml (hunk 1 above)
Comment (w0rk3r) [reply rules/windows/lateral_movement_remote_file_copy_hidden_share.toml:41]: I am not sure if there is any benchmark comparing them. but shouldn't cause a lot of impact as they need to match the first condition before the wildcard

**Review by Aegrah — APPROVED** [review]

**Pull request merged**